### DISCUSSION TITLE
Need help on syntax for editing project roles

### DISCUSSION THREAD

**Discussion by hrithik062** [langfuse/langfuse#2145] [Support]
I have a self hosted langfuse setup and as a owner of the project. I need to make the members in my project dynamically through python code rather than assigning them one by one in ui. Is there any syntax or way to bypass this

**Comment by marcklingen** [comment]
That's currently not possible yet. If you self-host, you could manually add to the `project_roles` table but that is discouraged as there are some planned changes which will break this. How many users do you plan to add?

**Comment by hrithik062** [comment]
I need to add around 400 ids as viewers and since we are using it in
organisation, There would be cases where we need to deleted like 30 or 40
and adding another of these ids. So if there's any flexibility around it,
it would be good

**Reply by marcklingen** [reply]
you could set up the project as the default project. thereby all users who newly sign up get default access to the project. is this a potential solution?
